dvc/repo/experiments/executor/base.py

```diff
@@ -540,11 +543,17 @@ def filter_pipeline(stages):
                 info.name,
                 repro_force or checkpoint_reset,
             )
+
+            def after_repro():
+                dvc.scm_context.track_file(cls._get_top_level_paths(dvc))
+                dvc.scm_context.track_changed_files()
```

Comment:
@daavoo, I am going to change this to use `dvc.scm.add()` directly, as `scm_context.track_file()` will make it show up in the message even if it has not changed.